napari/components/viewer_model.py

```diff
@@ -188,6 +188,7 @@ class ViewerModel(KeymapProvider, MousemapProvider, EventedModel):
     help: str = ''
     status: Union[str, dict] = 'Ready'
     tooltip: Tooltip = Field(default_factory=Tooltip, allow_mutation=False)
+    target_fps: tuple[int, int] = (30, 60)
```

Comment:
```suggestion
    target_fps: tuple[int, int] = (30, 59)
```
Not sure if it's hardware related or what, but it feels like fps is limited to 60 regardless of this PR. By dropping to 59 (or even a bit lower), lets the system find a better compromise of quality to speed?

Reply:
Yeah, probably depends on your monitor. But I don't know whether we want to keep increasing the quality if we are at 60 (or close to 60) — that means you can never really have a buttery-smooth framerate, we'll always "be greedy" and go for more quality, even when not necessary. I think I'd rather keep this working on the low-end, and only deal with the high end in subsequent PRs. Ultimately, there's probably a lot of control theory to take into account to deal with this optimally...

Reply:
Right, but you're dropping the quality by a factor of 4, so could easily be overshooting?

Reply:
We can certainly fiddle with the adjustment rates... I could switch to 2 for now, then in future updates, we can do some fancy math based on the difference between the current fps and the final fps. Actually, it would be interesting to make the quality change numerical, rather than just up or down. But I think this should come in a later PR. For now I think 2x is probably a good heuristic, even if it takes a bit longer to get to the target.

Reply:
I think with how high the quality goes on stop moving, the 4x is good, gets things moving again quicker.

Reply:
(That said, I can't really see a difference between 0.1 and 0.5 on the data I'm looking at)